test/unit/resources/wmi_test.rb

```diff
@@ -24,6 +24,7 @@
     resource = MockLoader.new(:ubuntu).load_resource("wmi", { class: "win32_service", filter: "name like '%winrm%'" })
     _(resource.resource_failed?).must_equal true
     _(resource.resource_exception_message)
-      .must_equal "Resource `wmi` is not supported on platform ubuntu/20.04."
+      .must_equal "Resource `wmi` is not supported on platform ubuntu/22.04."
+    _(resource.resource_id).must_equal "win32_service"
```

Comment:
This test should not go in the inspec-4 codebase - since inspec-4 has no support for resource_ids, no need to test it.